src/btree/bt_read.c

```diff
@@ -190,6 +190,9 @@ __las_page_instantiate(WT_SESSION_IMPL *session, WT_REF *ref)
 		upd->txnid = las_txnid;
 		upd->timestamp = las_timestamp;
 		upd->prepare_state = prepare_state;
+		/* Copy the commit timestamp as durable timestamp. */
```

Comment:
I guess this is OK for now, since we aren't writing prepared updates to lookaside, it'll always be correct. I think it's worth expanding the comment to reflect that:

Use the commit timestamp as the durable timestamp, since prepared updates don't currently get written to lookaside, so the two timestamps should always be identical.